src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+
+  for (const key in robot) {
+    if (newRobot[robot[key]] !== undefined) {
```

Comment:
Consider using 'Object.prototype.hasOwnProperty.call()' to check for properties directly on the object, rather than checking for undefined values.